satpy/etc/readers/seviri_l2_grib.yaml

```diff
@@ -73,244 +73,284 @@ datasets:
 
   cloud_mask:
     name: cloud_mask
+    long_name: Cloud Classification
+    standard_name: cloud_classification
     resolution: 3000.403165817
     file_type: grib_seviri_clm
     parameter_number: 7
     units: "1"
-    long_name: cloud_mask_classification
+    flag_values: [0, 1, 2, 3]
+    flag_meanings: ['clear sky over water','clear sky over land', 'cloudy', 'no data' ]
 
   pixel_scene_type:
     name: pixel_scene_type
+    long_name: Scene Type Classification
+    standard_name: scene_classification
     resolution: 3000.403165817
     file_type: grib_seviri_oca
     parameter_number: 8
     units: "1"
-    long_name: scene_classification
+    flag_values: [0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,50,97,98,99,100]
+    flag_meanings: ['none','evergreen needleleaf forest','evergreen broadleaf forest', 'decidious needleleaf forest','decidious broadleaf forest','mixed forest','closed shrubland','open shrubland','woody savannahs','savannahs','grasslands','permananent wetlands','croplands','urban and build-up','cropland mosaics',' permanent snow/ice','bare soil and rocks','water bodies','tundra','mixed land/water','unknown scene','snow/ice over land','snow/ice over watter','clear sunglint','cloudy']
```

Comment:
This is false and should be the OCA scene classification including information about liquid water clouds, ice clouds and two-layer clouds.

Reply:
corrected
Changed long_name / standard_name to Cloud Type / cloud_type